src/instrumenter/annotations.ts

```diff
@@ -234,42 +275,27 @@ export class AnnotationExtractor {
                 annotationOrig,
                 errRange
             );
-        }
-
-        try {
-            const exprNode = parseExpr(exprOrig);
-
-            return new Annotation(
+        }*/
+        if (parsedAnnot instanceof SProperty) {
+            return new PropertyMD(meta.node, meta.target, annotationOrig, parsedAnnot, match.index);
```

Comment:
Could an "early `return`s" be used here to reduce nesting?

Reply:
Maybe for the last else case? (`throw new Error(`NYI annotation ${parsedAnnot.pp()}`);`)? But I kinda like having all 3 on the same level here, to show that they are parts of a switch